techniques/general/G225.html

```diff
@@ -96,15 +96,15 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
       
       
       <section id="description">
-         <h2>Description</h2>
-         <p>The objective of this technique is to ensure that users can zoom in on text within a horizontally scrolling section, while not requiring horizontal scrolling to read the content of each individual panel. Although the entire section requires horizontal scrolling to navigate between panels, each panel is designed to fit within a fixed width of 320 CSS pixels. This ensures that when a 320 CSS pixel wide viewport is used, and the card is in the viewport, each card will remain fully visible without the need for additional horizontal scrolling to read its content.</p>
+         <h2>解説</h2>
+         <p>このテクニックの目的は、水平にスクロールするセクション内のテキストを利用者が拡大表示できるようにづる一方で、各パネルの内容を読むために水平スクロールを必要としないことを保証することである。セクション全体ではパネル間をナビゲートするために水平スクロールが必要であるが、各パネルは 320 CSS ピクセルの固定幅に収まるように設計されている。これにより、320 CSS ピクセル幅のビューポートを使用して、カードがビューポート内にある場合でも、各カードの内容を読むために追加の水平スクロールを必要とせずに、カード全体が常に表示される。</p>
       </section>
       <section id="examples">
-         <h2>Examples</h2>
+         <h2>事例</h2>
          <section class="example" id="example-1-a-product-carousel">
-             <h3>Example 1: A product carousel</h3>
-             <p>The homepage of an online retail website includes several informational sections, some of which feature carousels showcasing highlighted products. These carousels require horizontal scrolling and have a computed height greater than 256 CSS pixels. Each panel is 320 CSS pixels wide, allowing users to see the full width of the card without horizontal scrolling. Consequently, users only need to scroll vertically to view the entire card, ensuring compliance with success criterion 1.4.10 Reflow for that section.</p>
-             <p>Working example: <a href="../../working-examples/reflow-carousel-panel-horizontal-scroll/">Carousel panels that scroll horizontally are designed to fit within a width of 320 CSS pixels on a vertically scrolling page</a>.
+             <h3>事例 1: 製品のカルーセル</h3>
+             <p>あるオンライン小売ウェブサイトのホームページには、複数の情報セクションが含まれており、その中には注目の製品を紹介するカルーセルが配置されている。これらのカルーセルは水平スクロールを必要とし、計算高さは 256 CSS ピクセルを超えている。各パネルの幅は 320 CSS ピクセルで、利用者は水平スクロールすることなくカードの全幅を見ることができる。そのため、利用者は垂直スクロールだけでカード全体を閲覧でき、このセクションは達成基準 1.4.10 リフローを遵守している。</p>
```

Comment:
「computed height」の訳を、調整してみました。

```suggestion
             <p>あるオンライン小売ウェブサイトのホームページには、複数の情報セクションが含まれており、その中には注目の製品を紹介するカルーセルが配置されている。これらのカルーセルは水平スクロールを必要とし、計算される高さは 256 CSS ピクセルを超えている。各パネルの幅は 320 CSS ピクセルで、利用者は水平スクロールすることなくカードの全幅を見ることができる。そのため、利用者は垂直スクロールだけでカード全体を閲覧でき、このセクションは達成基準 1.4.10 リフローを遵守している。</p>
```